File: Third_Year/DS/Practicas/P1/Filtros/Alquiler.cpp
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
#include "iostream"
using namespace std;
#include "Alquiler.h"


/***** METODOS GET *****/
float Alquiler::getPrecio(){
    return precio;
}

string Alquiler::getDeporte(){
    return deporte;
}

int Alquiler::getTiempo(){
    return tiempo;
}

int Alquiler::getPersonas(){
    return n_personas;
}

/***** METODOS GET *****/
void Alquiler::setPrecio(float coste){
    precio = coste;
}

void Alquiler::setDeporte(string dep){
    deporte = dep;
}

void Alquiler::setTiempo(int tmp){
    tiempo = tmp;
}

void Alquiler::setPersonas(int n){
    n_personas = n;    
}

void Alquiler::setAlquiler(float coste, string dep, int tmp, int n){
    precio = coste;
    deporte = dep;
    tiempo = tmp;
    n_personas = n;
}

File: Third_Year/DS/Practicas/P1/Filtros en Dart/Alquiler.dart
class Alquiler{
  double _precio;
  int _tiempo, _n_personas;
  String _deporte;

  double getPrecio(){
    return _precio;
  }

  String getDeporte(){
    return _deporte;
  }

  int getTiempo(){
    return _tiempo;
  }

  int getPersonas(){
    return _n_personas;
  }

  void setPrecio(double coste){
    _precio = coste;
  }

  void setDeporte(String dep){
    _deporte = dep;
  }

  void setTiempo(int tmp){
    _tiempo = tmp;
  }

  void setPersonas(int n){
    _n_personas = n;
  }

  void setAlquiler(double coste, String dep, int tmp, int n){
    _precio = coste;
    _deporte = dep;
    _tiempo = tmp;
    _n_personas = n;
  }
}